src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/PipelineAttacher.java

```diff
@@ -0,0 +1,211 @@
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                        if (!globs.isEmpty()) {
+                            LOGGER.log(Level.FINE, "seem to be recording {0} in {1}", new Object[] {globs, job});
+                            run.addAction(new PipelineRealtimeTestResultAction(node2.getId(), workspace, keepLongStdio, globs));
+                            AbstractRealtimeTestResultAction.saveBuild(run);
+                        } else {
+                            LOGGER.log(Level.FINE, "no junit recorded in {0}", run);
+                        }
+                    }
+                } else {
+                    LOGGER.finer("no WorkspaceAction");
+                }
+            }
+        } else if (node instanceof BlockEndNode) {
+            BlockStartNode startNode = ((BlockEndNode) node).getStartNode();
+            if (startNode.getPersistentAction(WorkspaceAction.class) == null) {
+                LOGGER.finer("no WorkspaceAction");
+                return; // shortcut
+            }
+            String startNodeId = startNode.getId();
+            Queue.Executable executable;
+            try {
+                executable = node.getExecution().getOwner().getExecutable();
+            } catch (IOException x) {
+                LOGGER.log(Level.WARNING, null, x);
+                return;
+            }
+            if (executable instanceof Run) {
+                Run<?, ?> run = (Run<?, ?>) executable;
+                for (PipelineRealtimeTestResultAction a : run.getActions(PipelineRealtimeTestResultAction.class)) {
+                    if (a.startNodeId.equals(startNodeId)) {
+                        run.removeAction(a);
+                        LOGGER.log(Level.FINE, "clearing {0}", node);
+                        AbstractRealtimeTestResultAction.saveBuild(run);
+                        break;
+                    }
+                }
+            } else {
+                LOGGER.log(Level.WARNING, "not a Run: {0}", executable);
+            }
+        }
+    }
+
+    private static class PipelineRealtimeTestResultAction extends AbstractRealtimeTestResultAction {
+
+        private final String startNodeId;
+        private final String node;
+        private final String workspace;
+        private final boolean keepLongStdio;
+        private final Set<String> globs;
+
+        PipelineRealtimeTestResultAction(String startNodeId, FilePath ws, boolean keepLongStdio, Set<String> globs) {
+            this.startNodeId = startNodeId;
+            node = FilePathUtils.getNodeName(ws);
+            workspace = ws.getRemote();
+            this.keepLongStdio = keepLongStdio;
+            this.globs = globs;
+        }
+
+        @Override
+        public String getDisplayName() {
+            return "Realtime Test Result on " + (node.isEmpty() ? "master" : node);
```

Comment:
missing localization

Reply:
Well, the plugin had no localization before, but sure.